var/spack/repos/builtin/packages/petaca/package.py

```diff
@@ -0,0 +1,54 @@
+# Copyright 2013-2021 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack import *
+
+
+class Petaca(CMakePackage):
+    """A collection of modern Fortran modules.
+
+    Modules include:
+    - Parameter lists
+    - map_any_type
+    - fortran_dynamic_loader
+    - timer_tree_type
+    - yajl_fort
+    - json
+    """
+
+    homepage = "https://petaca.readthedocs.io/en/master"
+    git      = "https://github.com/nncarlson/petaca.git"
+
+    maintainers = ['pbrady']
+
+    version('develop', branch="master")
+    version('1.0', commit='f17df95193ca1a3879687a59a91a123be25e3efa', preferred=True)
+
+    depends_on('cmake@3.3:', type='build')
+    depends_on('yajl@2.0.1:')
+
+    variant("unit", default=False, description='build test programs')
+
+    variant("examples", default=False, description='build example programs')
```

Comment:
Since these don't affect the installed binaries, remove them. You can replace the args in your recipe:
```
self.define('ENABLE_TESTS', self.run_tests),
```
and hypothetically can replace the EXAMPLES with a "smoke test" (although the smoke test interface is unstable right now so I wouldn't recommend it.)

Reply:
Thanks @sethrj. I wasn't aware of the `self.run_tests` options.  I've removed the variants.